dask/dataframe/core.py

```diff
@@ -4230,16 +4237,25 @@ def sort_values(
         """
         from .shuffle import sort_values
 
+        if sort_function is None:
+            sort_function = M.sort_values
+        if sort_function_kwargs is None:
+            sort_function_kwargs = {
+                "by": by,
+                "ascending": ascending,
+                "na_position": na_position,
+            }
```

Comment:
hmm this would make a bit more sense to me if the `sort_function_kwargs` just updated this dict of default kwargs. Does that make sense? So that you could do something like:

```python
ddf.sort_values("col", sort_function_kwargs={"na_position": "first"})
```
```suggestion
       function_kwargs = {
            "by": by,
            "ascending": ascending,
            "na_position": na_position,
        }
        if function_kwargs is not None:
            function_kwargs.update(sort_function_kwargs)
```

Reply:
That makes sense! I can push that change in